authserver/db/models/models.py

```diff
@@ -227,7 +227,7 @@ class Meta:
     user_id = fields.String(required=True)
     client_id = fields.String(dump_only=True)
     client_secret = fields.String(dump_only=True)
-    issued_at = fields.Integer(dump_only=True)
+    client_id_issued_at = fields.Integer()
```

Comment:
@reginafcompton is there a reason why you don't want to enforce that this field is only of interest when we dump the output? I

Reply:
For posterity, we chatted, and Greg speaks the truth: we want to enforce `dump_only` (i.e., the field should not be override-able via POST).